numpy/random/generator.pyx

```diff
@@ -3361,6 +3361,19 @@ cdef class Generator:
         tol : float, optional
             Tolerance when checking the singular values in covariance matrix.
             cov is cast to double before the check.
+        method : { 'svd', 'eigh', 'cholesky'}, optional
+            The cov input is used to compute a factor matrix A such that
+            transpose(A) * A = cov. This argument is used to select the method
```

Comment:
should this be transpose(A).dot(A) or transpose(A) @ A?  `*` seems ambiguous especially since the matrix class is discouraged.

Reply:
Yeah, dot is probably better notation. I will fix it now